docs/bcfusion/validators/creation.md

```diff
@@ -1,2 +1,137 @@
+import Validator1 from '../../assets/bcfusion/create-validator1.png';
+import Validator2 from '../../assets/bcfusion/create-validator2.png';
+import Validator3 from '../../assets/bcfusion/create-validator3.png';
+import Validator4 from '../../assets/bcfusion/create-validator4.png';
+
+# Validator Creation
+
+If you are migrating old validators created on
+the BNB Beacon Chain, please refer to [this document](migrations.md) for more information.
+
+You can new the BNB staking dApp to create your validators.
+
+BNB Staking dApp on testnet: https://testnet-staking.bnbchain.org/en/bnb-staking
+
+BNB Staking dapp on mainnet: To be updated.
+
+After opening the dapp, you can click the `Become a Validator` button to create a new validator.
+
+The following information will be required in the `Create Validator` page.
+
+## Basic Info
+
+<img src={Validator1} width="400"/>
+
+- **Validator Name**: The name of your validator.
+- **Website**: Your website for sharing more information about your validator.
+- **Description**: Description of your validator.
+
+You can upload more information in the [BSC validator directory](https://github.com/bnb-chain/bsc-validator-directory).
+After you uploading the avatar in the repo, it will be displayed in the staking dApp.
+
+## Addresses
+
+<img src={Validator2} width="400"/>
+
+- **Consensus address**: The consensus address of the validator's node.
+- **Vote Address**: The address for participating fast finality voting.
+- **BLS Proof**: A BLS signature to prove that the validator owns the vote address.
+- **Identity**: If you are f you are a old validator operator on the BC side, please make a validator mapping signature
+  via the following steps.
+
+### Create vote address & BLS proof
+
+Please down load BSC geth binary here: https://github.com/bnb-chain/bsc/releases/tag/v1.3.10
+
+Create a BLS account or import an existing BLS account you already have.
+
+```shell
+ geth bls account new
+ geth bls account new
+```
+
+Then you can get your vote address by running the following command.
+
+```shell
+geth bls account list
+```
+
+An example address
+is `b5fe571aa1b39e33c2735a184885f737a59ba689177f297cba67da94bea5c23dc71fd4deefe2c0d2d21851eb11081f69`.
+
+Then you can get your bls proof by running the following command.
+
+```shell
+geth_macos bls account generate-proof --chain-id ${BSC_CHAIN_ID} ${OPEATOR_ADDRESS} ${VOTE_ADDRESS}
+```
+
+An example proof
+is `0xaf762123d031984f5a7ae5d46b98208ca31293919570f51ae2f0a03069c5e8d6d47b775faba94d88dbbe591c51c537d718a743b9069e63b698ba1ae15d9f6bf7018684b0a860a46c812716117a59c364e841596c3f0a484ae40a1178130b76a5`.
+
+- `BSC_CHAIN_ID`: 56 for BSC mainnet, and 97 for BSC testnet.
+- `OPEATOR_ADDRESS`: The address of your account, which will be reconginzed as the operator of the new validator.
+- `VOTE_ADDRESS`: The vote address created in the last step.
+
+### Create indentity
+
+Please down load BC client binary here: https://github.com/bnb-chain/node/releases/tag/v0.10.18
```

Comment:
please hightlight that only the old validator who want to migrate need to do this, and no need for new BSC validators.

Reply:
Sure. Added.